### PR TITLE
LPS-169348 Object Entries ignore parameters that are not defined

### PR DESCRIPTION
**Pull request by pedro-oliveira446**
[LPS-169348](https://issues.liferay.com/browse/LPS-169348)

### CODE REVIEW

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
Hi @pedro-oliveira446, there are some failures that seem to be unique. Could you check before I start the review? Thanks!

**Comment by pedro-oliveira446** [comment]
ci:test:sf

**Comment by pedro-oliveira446** [comment]
ci:test:relevant

**Comment by pedro-oliveira446** [comment]
ci:test:object

**Comment by pedro-oliveira446** [comment]
ci:test:sf

**Comment by pedro-oliveira446** [comment]
ci:test:relevant

**Comment by pedro-oliveira446** [comment]
ci:test:object

**Comment by pedro-oliveira446** [comment]
ci:test:relevant

**Comment by pedro-oliveira446** [comment]
ci:test:object

**Comment by pedro-oliveira446** [comment]
@carolmariaabb,
I made some changes and I believe that now, the failures are not related. But I saw that for some CI problem some tests didn't run, so I ran them again.

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
@pedro-oliveira446 could you please check if the failure in `com.liferay.site.initializer.extender.internal.test.BundleSiteInitializerTest ` is related to your changes? This test is part of the ci:stable suite and it creates object definitions and object entries.

**Comment by carolmariaabb** [comment]
@pedro-oliveira446 It seems to be related. Please check the error message:

image

**Comment by pedro-oliveira446** [comment]
ci:test:object

File: modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/manager/v1_0/DefaultObjectEntryManagerImpl.java [hunk 1]
```diff
@@ -1268,6 +1282,81 @@ else if (value instanceof ListEntry) {
...
+								objectLayoutBox.getObjectLayoutBoxId()) {
+
+							return true;
+						}
+					}
+				}
+			}
+		}
+
+		return false;
```
Comment (carolmariaabb) [inline modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/manager/v1_0/DefaultObjectEntryManagerImpl.java:1311]: @pedro-oliveira446 some improvements here:

Buy space:

```
if (!Validator.isNumber(key)) {
	return false;
}
```
and inline:

```
for (ObjectLayout objectLayout : _objectLayoutLocalService.getObjectLayouts(objectDefinitionId)) {
	for (ObjectLayoutTab objectLayoutTab : objectLayout.getObjectLayoutTabs()) {
		for (ObjectLayoutBox objectLayoutBox : objectLayoutTab.getObjectLayoutBoxes()) {
			if (Long.valueOf(key) ==
					objectLayoutBox.getObjectLayoutBoxId()) {

				return true;
			}
		}
	}
}

return false;
```

**Comment by carolmariaabb** [comment]
I'll close this PR for now and discuss with the team about it.

**Pull request closed without merging**